OpenOversight/app/commands.py

```diff
@@ -162,17 +162,14 @@ def set_field_from_row(row, obj, attribute, allow_blank=True, fieldname=None):
 
 
 def update_officer_from_row(row, officer, update_static_fields=False):
-    def update_officer_field(fieldname, allow_blank=True):
+    def update_officer_field(fieldname):
         if fieldname not in row:
             return
-        if row[fieldname] == '':
-            row[fieldname] = None
 
         if fieldname == 'gender':
             row[fieldname] = normalize_gender(row[fieldname])
 
-        if fieldname in row and (row[fieldname] or allow_blank) and \
-                getattr(officer, fieldname) != row[fieldname]:
+        if row[fieldname] and getattr(officer, fieldname) != row[fieldname]:
```

Comment:
This fixes a bunch of failing `test_csv_xxx` tests that were a result of a merge with changes made on the develop-branch.

Reply:
so IIRC the specific behavior we wanted to fix with `update_officer_field` and allow_blank=False is that if the officer already exists, and they already have a non-null gender, we do not want a spreadsheet import with that officer's gender missing to cancel out that existing gender. Like, if IVANA TINKLE (F) is already in the database and she gets updated with a spreadsheet that matches her on name/DOB/badge number but has no gender, the most likely interpretation of that isn't that we are no longer aware of Ivana's gender, it's that whoever made the new spreadsheet didn't think to include gender in it. 

However, fixing the error so the tests pass is probably the best way to go, and then I can add another commit that 1) tests the behavior we want to see (don't overwrite existing genders due to a spreadsheet not having them) and 2) adds that behavior back in in a less convoluted way (I realize that if statement got real gnarly).

Reply:
That behavior is maintained in those changes. If gender is `None` the updating piece will be skipped because of the  `if row[fieldname]` condition. But yeah, definitely a good idea to cover that scenario with a test